Dockerfile

```diff
@@ -63,7 +63,7 @@ CMD ["/runtests.sh"]
 
 
 # Stage 4 - Build docker image suitable for execution and deployment
-FROM python:3.6-alpine AS production
```

Comment:
I think this is the only relevant change for this PR. Note that this will also require `bin/release_docker_image.sh` to be updated - the build target in https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus/blob/develop/bin/release-docker-image.sh#L15 will no longer resolve